frontend/src/metabase/components/Popover/Popover.module.css

```diff
@@ -53,7 +53,6 @@
 
 :global(.tippy-box[data-theme~="popover"]) {
   font-size: inherit;
-  color: var(--color-text-default);
```

Comment:
This custom property doesn't exist. Observing the devtool will see that the custom property is grayed out.
![Screenshot 2024-04-18 at 7 52 18 PM](https://github.com/metabase/metabase/assets/1937582/66cbbc41-cb4c-4946-8001-355e39fb0275)